File: php/examples/get_simple_products.php
<?php
require_once 'base_request.php';

// Example of GET /simple-products

$debug = true;
$website_url   = $_ENV['WEBSITE_URL'];
$request_data = '';
$request_params = ['url' => $website_url];

send_request('GET', '/simple-products', $request_params, $request_data);

File: csharp/examples/GetSimpleProducts.cs
using System;
namespace Examples.examples
{
    // Example of GET /simple-products

    public class GetSimpleProducts : BaseRequest
    {
        private string _websiteUrl;

        public GetSimpleProducts()
        {
            dotenv.net.DotEnv.Load();
            _websiteUrl = Environment.GetEnvironmentVariable("WEBSITE_URL");
            Console.WriteLine($"GetSimpleProducts: _websiteUrl={_websiteUrl}");
        }

        public override void SendRequest()
        {
            GetRequest($"simple-products?url={_websiteUrl}");
        }
    }
}

